gnomad/utils/vep.py

```diff
@@ -802,9 +802,69 @@ def filter_vep_transcript_csqs(
     vep_root: str = "vep",
     synonymous: bool = True,
     canonical: bool = True,
-    mane_select: bool = False,
-    filter_empty_csq: bool = True,
     ensembl_only: bool = True,
+    filter_empty_csq: bool = True,
```

Comment:
Thoughts on updating to `remove_empty_csq` ? 
```suggestion
    remove_empty_csq: bool = True,
```

Reply:
Looks like we'd only need to update gnomad_qc in one place and then there would be a bulk replace in your other VEP PR. I just think remove is more clear since filter is a bit overloaded and can be ambiguous as to whether you are keeping or removing the filtered material.

Reply:
I think it makes sense to do it in it's own PR where that is all that is done since it's not a name introduced in this PR, and it is technically a breaking change